### PR TITLE
Updated Help Menu

### LINKED ISSUE #8070: Update the Help Menu
Hello @N-thony,

Please could someone restructure the Help menu slightly?

Roger is going to provide the new helpfile and the correct numbers so that things link up correctly. 

We would like it to look like this. 

Thanks! Let me know if you have any questions.

### PR DESCRIPTION
**Pull request by Vitalis95**
Fixes #8070 
@rdstern @lloyddewit @N-thony ,  I changed the help menu as requested, The  **Glossary** and **R-Packages** doesn't have the **help file**. Once Roger provides them I will add. Thanks

### CODE REVIEW

**Comment by rdstern** [comment]
@Vitalis95 the help context for Glossary is 92
For R packages it is 26

**Review by rdstern — DISMISSED** [review]
@Vitalis95 this seems to work fine

File: instat/frmMain.vb [hunk 1]
```diff
@@ -102,7 +102,7 @@ Public Class frmMain
         SetMainMenusEnabled(False)
         Cursor = Cursors.WaitCursor
         'temporary
-        mnuHelpAboutRInstat.Visible = False
+        'mnuHelpAboutRInstat.Visible = False
```
Comment (lloyddewit) [inline instat/frmMain.vb:105]:
```suggestion
```

File: instat/frmMain.vb [hunk 2]
```diff
@@ -1178,11 +1178,11 @@ Public Class frmMain
     End Sub
 
 
-    Private Sub mnuHelpGuidesCaseStudy_Click(sender As Object, e As EventArgs) Handles mnuHelpGuidesCaseStudy.Click
+    Private Sub mnuHelpGuidesCaseStudy_Click(sender As Object, e As EventArgs)
```
Comment (lloyddewit) [inline instat/frmMain.vb:1181]: Is this sub used anywhere? can we delete it?

File: instat/frmMain.vb [hunk 3]
```diff
@@ -1178,11 +1178,11 @@ Public Class frmMain
...
         Process.Start(Path.Combine(strStaticPath, "Help", "Case_Study_Guide_June_2016.pdf"))
     End Sub
 
-    Private Sub mnuHelpGuideGlosary_Click(sender As Object, e As EventArgs) Handles mnuHelpGuideGlosary.Click
+    Private Sub mnuHelpGuideGlosary_Click(sender As Object, e As EventArgs)
```
Comment (lloyddewit) [inline instat/frmMain.vb:1185]: Is this sub used anywhere? can we delete it?

File: instat/frmMain.vb [hunk 4]
```diff
@@ -1941,10 +1941,12 @@ Public Class frmMain
         dlgFitModel.ShowDialog()
     End Sub
 
-    Private Sub mnuHelpAcknowledgments_Click(sender As Object, e As EventArgs) Handles mnuHelpAcknowledgments.Click
+    Private Sub mnuHelpAcknowledgments_Click(sender As Object, e As EventArgs)
```
Comment (lloyddewit) [inline instat/frmMain.vb:1944]: Is this sub used anywhere? can we delete it?

File: instat/frmMain.vb [hunk 5]
```diff
@@ -1941,10 +1941,12 @@ Public Class frmMain
...
         Help.ShowHelp(Me, strStaticPath & "\" & strHelpFilePath, HelpNavigator.TopicId, "151")
     End Sub
-
+    Private Sub mnuHelpGlossary_Click(sender As Object, e As EventArgs) Handles mnuHelpGlossary.Click
+        Help.ShowHelp(Me, strStaticPath & "\" & strHelpFilePath, HelpNavigator.TopicId, "92")
+    End Sub
```
Comment (lloyddewit) [inline instat/frmMain.vb:1949]: separate subs with empty line
```suggestion
    End Sub
    
```

File: instat/frmMain.vb [hunk 6]
```diff
@@ -1996,7 +1998,7 @@ Public Class frmMain
         Help.ShowHelp(Me, strStaticPath & "\" & strHelpFilePath, HelpNavigator.TopicId, "539")
     End Sub
 
-    Private Sub mnuHelpDataViewSpreadsheet_Click(sender As Object, e As EventArgs) Handles mnuHelpDataViewSpreadsheet.Click
+    Private Sub mnuHelpDataViewSpreadsheet_Click(sender As Object, e As EventArgs)
```
Comment (lloyddewit) [inline instat/frmMain.vb:2001]: Is this sub used anywhere? can we delete it?

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Comment by Vitalis95** [comment]
@lloyddewit , have a look at it

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern there are some small changes since your last approval. If you can retest/approve, then we can merge, thanks

**Review by rdstern — APPROVED** [review]

**Pull request merged**